R/bs-theme.R

```diff
@@ -128,11 +128,15 @@ bs_theme <- function(version = version_default(), preset = NULL, ...,
   preset <- resolve_bs_preset(preset, bootswatch, version = version)
 
   bundle <- bs_bundle(
-    bs_theme_init(version, subclass = preset$class),
+    bs_theme_init(version),
     bootstrap_bundle(version),
     bs_preset_bundle(preset)
   )
 
+  if (!is.null(preset$type)) {
+    bundle <- add_class(bundle, "bs_theme_with_preset")
```

Comment:
Let's store `"bs_theme_with_preset"` in a reusable variable?

Reply:
In that case, I think we should store it in the `preset` object

Reply:
Restored in 1a602d265efde136e5e6504318f366d696ca6266